src/Agent.Sdk/SecretMasking/LoggedSecretMasker.cs

```diff
@@ -11,6 +15,8 @@ public class LoggedSecretMasker : ILoggedSecretMasker
         private ISecretMasker _secretMasker;
         private ITraceWriter _trace;
 
+        public bool UseCompiledRegex { get; set; } = false;
```

Comment:
>UseCompiledRegex [](http://example.com/codeflow?start=20&length=16)

This should be a change to something like 'UsePerformanceEnhancements', yes? In other words, you are introducing this boolean to allow for conditional use of all the proposed performance changes.

Reply:
@michaelcfanning, renamed, thanks.